### PR TITLE
Milestone 1 : Using Bootstrap (Newsweek) 

### PR DESCRIPTION
**Pull request by ZeenatLawal**
## Newsweek clone milestone 1: Using Bootstrap

In this milestone we:

- Added two sections to the site (Header and Footer)
- Used bootstrap's default media query breaking points
- Used bootstrap classes for the styling

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Status: Change Requested

Amazing job so far @ZeenatLawal and @safafa you implemented almost all of the requirements for this project but you need to make some small changes to be ready for the next milestone
## To Highlights image


- PR is professional
- Codes looks clean and easy to read
- README is descriptive

### Header

- The Newsweek logo has a link
- Logo In desktop it is centered
- In tablet and mobile, it is aligned to the left
- Contains a date in the same format on the source page (only on the desktop)
- Login and Subscribe have links
-  In mobile, the Login button changes into an icon
- 
### The main navigation (menu)
  * [ ] Uses a nav tag
  * [ ] Each option has a link
  * [ ] It has a form with the search field
  * [ ] In mobile and table the menu hides, and show a hamburger button
* [ ] The size of the elements change in desktop, tablet, and mobile

## To Improve image

### Header
- The header has to fit the content on the mobile 

original

original

yours

h1
Try to make it looks like the original

### Footer
- Newsweek logo is supposed to have a link
- The content-disposition changes to the correct place in desktop, tablet, and mobile

yours

copy

Original

og

### Kindly fix that and submit another CRR and remember you still have only 2 code reviews and work accordingly

All the best,

Lydia Ingabire

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Business</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Tech & Science</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Culture</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Newsgeek</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Sports</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Health</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">The Debate</a>
+            <a class="nav-link border-end link-dark fw-bold text-decoration-none" href="#">Experts</a>
+            <a class="nav-link link-dark fw-bold text-decoration-none" href="#">Vantage</a>
+            <a class="nav-link disabled" href="#">Search <i class="fas fa-search"></i></a>
```
Comment (teekaytech) [inline index.html]: This should be a form with the search field and not just the search icon.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @ZeenatLawal & @safafa  :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- Almost all the required changes have been fixed :+1: 
- Codebase looks clean with a good naming structure :1st_place_medal: 
- Limited use of custom CSS rules :ok_hand: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_**Note that you are asked to be more precise and to make sure that your page looks exactly the same as the source page**_

- [x] The red background color at the top navbar is not the same as what we have on the original page. Since the color is not the same as the `danger` color of bootstrap, it is okay to use a custom CSS rule to specify the correct color. 
- [x] Check my inline comment below and fix it as appropriate.

**[ OPTIONAL CHANGES ]**

- [ ] You've used the `id` attribute extensively as your CSS selectors. This will work, but it is not a very good CSS practice. You might want to change the `id` selectors to `class` selectors.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second-opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Elbie-em — APPROVED** [review]
## STATUS: APPROVED :heavy_check_mark:

Hello @safafa and @ZeenatLawal, good job implementing changes requested by the previous TSE and I will therefore approve this milestone.

## [OPTIONAL]

- Please ensure you apply the changes below requested in your first review accordingly before moving on.
     - [ ]  Your nav in the mobile version should look like the original. Apply changes to your news week logo, subscribe button, and spacing between elements in your navbar.

**Orginal** 
         
Screenshot 2021-04-06 at 11 01 59
      **Yours**
Screenshot 2021-04-06 at 11 25 05

 - [ ]  Please make sure your content-disposition changes to the correct place on desktop, tablet, and mobile. This fix has not been applied and I recommend you do it.
 
**Orginal**
Screenshot 2021-04-06 at 11 00 45

**Yours**
Screenshot 2021-04-06 at 11 25 17

*If you have any questions, you can reach me on slack using Elbie Moonga*

**Pull request merged**